aim/db/migration/alembic_migrations/versions/4caed435b0cd_add_netflow.py

```diff
@@ -58,6 +59,47 @@ def upgrade():
         sa.UniqueConstraint('name', name='uniq_aim_infra_identity'),
         sa.PrimaryKeyConstraint('aim_id'))
 
+    op.create_table(
+        'aim_vmm_vswitch_pol_grp',
+        sa.Column('aim_id', sa.String(64), nullable=False),
+        sa.Column('domain_type', sa.String(64), nullable=False),
+        sa.Column('domain_name', sa.String(64), nullable=False),
+        sa.Column('display_name', sa.String(256), nullable=False, default=''),
+        sa.Column('monitored', sa.Boolean, nullable=False, default=False),
+        sa.Column('epoch', sa.BigInteger(), nullable=False,
+                  server_default='0'),
+        sa.PrimaryKeyConstraint('aim_id'),
+        sa.UniqueConstraint('domain_type', 'domain_name',
+                            name='uniq_aim_vswitch_pol_grp_identity'),
+        sa.Index('idx_aim_vswitch_pol_grp_identity',
+                 'domain_type', 'domain_name'),
+        sa.ForeignKeyConstraint(
```

Comment:
We actually don't want to have foreign keys here - an AIM design principle allows for configuration of a resource in the DB even if the parent object hasn't yet been configured (and this reminds me that we need to fix the QoS patch, as it also introduced ForeignKey relationships which aren't needed). The exception is when a property maps to a DB table but not an AIM resource (see SecurityGroupRuleRemoteIp).

Reply:
That makes sense. I have made changes in the new commit.